recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "swifter" %}
-{% set version = "1.0.9" %}
+{% set version = "1.1.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 14cec10ff1ecdfbcfdd68985366b07add5f9b7bb77116720f47c88f29f048c3e
+  sha256: 7f49d49c8edf3661ed3c0db3d19896b539130330570654a442b6e426247d54b7
```

Comment:
updated the sha256 from `openssl sha256 swifter-1.1.2.tar.gz`

Reply:
I only have time tomorrow to look at the rest of the PR but according to the logs you have a SHA mismatch here.

The sea to use is `b954f3adb894f6b158ae86f4a2bdd44d8d52b75032f997f20a3c8c0b0da58fe4` as can also be seen on https://pypi.org/project/swifter/#copy-hash-modal-a9f3ec52-4764-4ab2-98f9-a8ed9b3eb80a

Reply:
Oh wow, thank you so much Bastian - i didnt know you could look at the hash through pypi directly. I really appreciate the hands-on assistance. Just updated the PR and hopefully it passes 🤞

Reply:
No worries! For the future, you can also get the SHA by running 
```zsh
curl -sL https://url.com/you/want/to/get.tar.gz | openssl sha256
```